Sources/PointFree/Account/Cancel.swift

```diff
@@ -34,7 +37,9 @@ private func cancelResponse(
     }
     _ = try await Current.stripe
       .cancelSubscription(stripeSubscription.id, stripeSubscription.status == .pastDue)
-    Task { try await sendCancelEmail(to: user, for: stripeSubscription) }
+    await fireAndForget {
+      try await sendCancelEmail(to: user, for: stripeSubscription)
+    }
```

Comment:
Here's an example of using the `fireAndForget` dependency. This makes it easy to test sending the cancellation email, which I added test coverage for.